requirements.txt

```diff
@@ -1,10 +1,13 @@
 # numpy<1.20
 pytz
 requests
-astropy==5.0
-numba
+scipy<1.9
+astropy
+numba<0.60
+matplotlib<3.9
```

Comment:
If you still have the notes from the failed runs, please document here in comments why each of these version constraints is needed.  e.g. is it directly due to specsim itself, or incompatibilities in the available 3rd party libraries that aren't fully specified in their own constraints?

Reply:
I added some notes in the latest commit, please take a look.